jwt/jwt.go

```
@@ -182,7 +182,14 @@ func JWTMiddleware() func(req events.APIGatewayProxyRequest) (events.APIGatewayP
 			return handleMiddlewareResponse(http.StatusInternalServerError, "Internal server error")
 		}
 
-		cookie := req.Headers["Cookie"]
+		cookie := ""
```

went through entire function and found below issues

1. This function returns a function with three return values. But your handleMiddlewareResponse only returns two values.
2. use below code to parsing cookies from the header.
```
header := http.Header{}
for k, v := range req.Headers {
	header.Set(k, v)
}
reqCookies := header["Cookie"]

```

3. You are ignoring the error here. If LoadEnvConfig() fails, it might return empty/default config which could result in missing cookieName.

---

will handle this as a part of this https://github.com/Real-Dev-Squad/feature-flag-backend/issues/158